build/installer_linux.sh

```diff
@@ -95,9 +95,12 @@ elif command -v 7zr &> /dev/null; then
 elif command -v 7za &> /dev/null; then
     # CentOS/Fedora
     COMMAND_7Z=7za
+elif command -v 7zz &> /dev/null; then
+    # Upstream
+    COMMAND_7Z=7zz
```

Comment:
`Upstream`だと意図が伝わりづらいかもなので、こうしますか！
```suggestion
elif command -v 7zz &> /dev/null; then
    # Official 7zip
    COMMAND_7Z=7zz
```

（意図と違ってたらコメントいただければ！）